stdlib/3/unittest/mock.pyi

```diff
@@ -92,7 +92,11 @@ if sys.version_info >= (3, 3):
         def start(self) -> Any: ...
         def stop(self) -> Any: ...
 
-    def patch(target: Any, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> Any: ...
+    class _patcher:
+        def __call__(self, target: Any, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> Any: ...
+        def object(self, target: Any, attribute: str, new: Any =..., spec: Any = None, create: Any = False, spec_set: Any = None, autospec: Any = None, new_callable: Any = None, **kwargs: Any) -> _patch: ...
```

Comment:
You're using `= None` in some places, and `= ...` in others.

I generally prefer the latter, but at the very least, it should be consistent within the same file.